airbyte-integrations/connectors/source-hardcoded-records/pyproject.toml

```diff
@@ -17,14 +17,16 @@ include = "source_hardcoded_records"
 
 [tool.poetry.dependencies]
 python = "^3.11,<3.13"
-airbyte-cdk = "^6.37"
+airbyte-cdk = "6.45.1.post46.dev14423672753"
```

Comment:
TODO: Bump to stable before merge. Pending:

- https://github.com/airbytehq/airbyte-python-cdk/pull/349